tests/v2/test_jsonrpc_waku.nim

```diff
@@ -2,15 +2,17 @@ import
   std/[unittest, options, sets, tables, os, strutils, sequtils],
   stew/shims/net as stewNet,
   json_rpc/[rpcserver, rpcclient],
+  eth/[keys, rlp], eth/common/eth_types,
   libp2p/[standard_setup, switch, multiaddress],
   libp2p/protobuf/minprotobuf,
   libp2p/stream/[bufferstream, connection],
   libp2p/crypto/crypto,
   libp2p/protocols/pubsub/pubsub,
   libp2p/protocols/pubsub/rpc/message,
+  ../../waku/v1/node/rpc/hexstrings,
   ../../waku/v2/waku_types,
   ../../waku/v2/node/wakunode2,
-  ../../waku/v2/node/jsonrpc/[jsonrpc_types,store_api,relay_api,debug_api,filter_api,admin_api],
+  ../../waku/v2/node/jsonrpc/[jsonrpc_types,store_api,relay_api,debug_api,filter_api,admin_api,private_api],
```

Comment:
At this point it may be nice to have this multi-line of just import everything in there, I think you can do that with a `*`

Reply:
Or export from one jsonrpc package I suppose

Reply:
After experimenting with imports/exports I changed this to a multiline import and had the API modules export their types (in 577b7a7). I'll also create a separate test module for each API, which should hopefully clean things up even further.